vhosting/vhosting_check.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+
+import argparse
+import urllib.request as req
+import os, re, requests, sys
+
+banners_dir = "./banners/"
```

Comment:
I don't think we'd want to do that, maybe try to use the ones directly out of ocfweb perhaps?